salt/utils/thin.py

```diff
@@ -15,6 +15,7 @@
 import tempfile
 import zipfile
 
+import Cryptodome
```

Comment:
Doesn't Cryptodome have C extension code in it?
Is it only python code interfacing with libraries on the system?

We can't send compiled code in the thin(at least normally, we can't)

Reply:
Agreed, this is just to keep our existing ssh tests happy until we add relenv support.

Reply:
I mis-understood why this broke. Previously the thin dir wouldn't include any crypto libraries and `salt/crypt.py` wouldn't cause any import error because all of the crypto imports were gated. We should be able to keep salt ssh working as it was before by making sure the crytography import is gated. I am working on fixing this.